client/src/MessageInboxView.tsx

```diff
@@ -192,7 +195,7 @@ function MessageInboxView(): JSX.Element {
                 Re: {selectedTransaction.asset.title}
               </Typography>
               {messages?.map((m) => (
-                <MessageCard message={m} isCurrentUser={m.user.id === user?.id} />
+                <MessageCard message={m} isCurrentUser={m.user.id === user?.user.id} />
```

Comment:
so i know this was my bad but this `user.user` is ugly
let's undo this change and make the change in auth.service to be
`return this.jwtService.sign(user, { secret: jwtConstants.secret });`
and that SHOULD still work. lmk if not